helper-cli/src/main/kotlin/utils/PathExcludeGenerator.kt

```diff
@@ -204,6 +203,7 @@ private val PATH_EXCLUDE_REASON_FOR_FILENAME = listOf(
     "*_test.go" to TEST_OF,
     "*coverage*.sh" to BUILD_TOOL_OF,
     ".editorconfig" to PathExcludeReason.OTHER,
+    ORT_REPO_CONFIG_FILENAME to BUILD_TOOL_OF,
```

Comment:
FWIW, I deliberately did not use alphabetic sorting in 929b73481a4ff1a953ac5be4fa321acebb86f51e for this special entry to make it stand out.